dll/directx/wine/dpnhpast/dpnhpast.rgs

```diff
@@ -0,0 +1,15 @@
+HKCR
+{
+    NoRemove Interface
+    {
+    }
```

Comment:
Why this empty block?

Reply:
Removed that file since it's not needed.